pkg/action/action_handler.go

```diff
@@ -144,6 +168,32 @@ func (h *ActionHandler) ExecuteAction(actionExecutionDTO *proto.ActionExecutionD
 	return result, nil
 }
 
+func keepAlive(tracker sdkprobe.ActionProgressTracker, stop chan struct{}) {
+
+	//TODO: add timeout
+	go func() {
+		var progress int32 = 0
+		state := proto.ActionResponseState_IN_PROGRESS
+
+		for {
+			progress = progress + 1
+			if progress > 99 {
+				progress = 99
+			}
+
+			tracker.UpdateProgress(state, "in progress", progress)
```

Comment:
Yea. It is currently for the only purpose to keep the action live, not showing the actual progress.